example/config

```diff
@@ -31,35 +32,22 @@ BR2_x86_64=y
 # BR2_xtensa is not set
 BR2_ARCH="x86_64"
 BR2_ENDIAN="LITTLE"
-BR2_GCC_TARGET_TUNE="generic"
-BR2_x86_generic=y
```

Comment:
`BR2_x86_generic` was a deprecated option and `BR2_GCC_TARGET_TUNE` was changed to be `nocona` (there is no more `generic`). I think these were the two most glaring changes the **could** possibly affect a BusyBox rebuild.